### PR TITLE
Dashboard page UI

### LINKED ISSUE #724: Implement dashboard page UI

### PR DESCRIPTION
**Pull request by sofiaromorales**
Screen Shot 2022-11-18 at 4 05 19 PM

### CODE REVIEW

File: srcv2/App.js [hunk 1]
```diff
@@ -3,16 +3,19 @@ import { withRouter, Redirect } from 'react-router-dom'
...
+import OnboardingContributorPage from './pages/OnboardingContributorPage'
 
 class App extends React.Component {
     render() {
         return (
             <div className='App'>
+                <Authentication/>
                 <div className='content pb-24'>
                     <PublicRoute
                         path='/'
```
Comment (rishkwal) [inline srcv2/App.js:21]:
```suggestion
                <div className='content pb-24'>
                    <PublicRoute
                        restricted
                        path='/'
```

Comment (stark-3k) [inline srcv2/App.js]: Make the `LandingPage` and `LoginPage` routes restricted
```suggestion
                <div className='content pb-24'>
                    <PublicRoute
                        restricted
                        path='/'
```

File: srcv2/App.js [hunk 2]
```diff
@@ -29,8 +32,17 @@ class App extends React.Component {
                         path='/dashboard'
                         component={DashboardPage}
                     />
```
Comment (stark-3k) [inline srcv2/App.js]: Redirect base url to `DashboardPage` when logged in
```suggestion
                        path={['/dashboard','/']}
                        component={DashboardPage}
                    />
```

File: srcv2/components/Selector.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+const Selector = ({ title, renderOptions, openOptions, setOpenOptions, loadingOptions }) => {
+    return (
+        <div>
+            <button type='button' className='border border-light rounded-lg px-4 py-1 w-full' onClick={() => setOpenOptions(!openOptions)}>
+                <div className='grid grid-flow-col auto-cols-max flex justify-between'>
+                    <p className='text-light text-left'>{title}</p>
```
Comment (stark-3k) [inline srcv2/components/Selector.js]: NIT: Light text is not blending well with the white background
```suggestion
                    <p className='text-black text-left'>{title}</p>
```

File: srcv2/components/PlanFundingOnboarding.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                        type='text'
+                        placeholder='Budget range'
+                        onChange={(e) => setBudgetRange(parseInt(e.target.value, 10))}
+                        className='
+                        form-control
+                        block
+                        w-full
+                        px-3
+                        py-1.5
+                        text-light
```
Comment (stark-3k) [inline srcv2/components/PlanFundingOnboarding.js]: NIT: Light text is not blending well with the white background
```suggestion
                        text-black
```

**Review by stark-3k — APPROVED** [review]

File: server.js [hunk 5]
```diff
@@ -134,9 +134,9 @@ app.get('/api/oauth-redirect', (req, res) => { //redirects to the url configured
         })
         .then((contributorInfo) => {
             if (contributorInfo.newUser) {
-                return res.redirect(`${SITE_ROOT}/onboarding`)
+                return res.redirect(`${SITE_ROOT}/${process.env.OAUTH_REDIRECT_NEW_USERS}`)
             }
-            res.redirect(`${SITE_ROOT}/dashboard`)
+            res.redirect(`${SITE_ROOT}/${process.env.OAUTH_REDIRECT_EXISTING_USERS}`)
```
Comment (stark-3k) [inline server.js:139]: Update `.env.example` with `OAUTH_REDIRECT_NEW_USERS` and `OAUTH_REDIRECT_EXISTING_USERS`

**Pull request merged**